src/automations/components/AutomationActionDescriptionSendNotification.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="automation-action-description-send-notification" v-bind="$attrs">
+    Send {{ indefiniteArticle() }} {{ blockDocument?.blockType.name.toLocaleLowerCase() ?? '' }} notification using
+    <BlockIconText :block-document-id="blockDocumentId" />
+    {{ recipients }}
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { computed } from 'vue'
+  import { AutomationActionSendNotification } from '@/automations/types/actions'
+  import { BlockIconText } from '@/components'
+  import { useBlockDocument } from '@/compositions'
+
+  defineOptions({
+    inheritAttrs: false,
+  })
```

Comment:
Is there a specific reason you don't want to inherit attrs?

Reply:
Ah I see its on `AutomationActionDescriptionRunDeployment` which is a fragment because it also renders a modal. When there are multiple root elements vues [fallthrough attributes](https://vuejs.org/guide/components/attrs.html#fallthrough-attributes) won't work automatically so you have to specify where they should fallthrough to. Which is done by disabling `inheritAttrs` and then binding `$attrs` to the element you want them to fall through to. 

In your case you have a single root element so you don't need to do anything special. So that can just be removed. Otherwise the `automation-action-description` class from `AutomationActionDescription.vue` won't end up being bound to your component.